cpp/ycm/ClangCompleter/Documentation.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2015 YouCompleteMe Contributors
+//
+// This file is part of YouCompleteMe.
+//
+// YouCompleteMe is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// YouCompleteMe is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with YouCompleteMe.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "Documentation.h"
+#include "ClangHelpers.h"
+
+#include <clang-c/Documentation.h>
+
+namespace YouCompleteMe {
+
+namespace {
+  bool CXCommentValid( const CXComment & comment ) {
```

Comment:
As you noticed in my test data, I might already *be* crazy. Anyway, yes, `4<<` done :)